zmon-common/src/main/java/org/zalando/zmon/domain/CheckDefinition.java

```diff
@@ -253,8 +266,23 @@ public String toString() {
         builder.append(lastModifiedBy);
         builder.append(", runtime=");
         builder.append(runtime);
+        builder.append(", tier=");
+        builder.append(tier);
         builder.append("]");
         return builder.toString();
     }
 
+    public enum Tier {
+        @JsonProperty("empty")
+        EMPTY,
+
+        @JsonProperty("critical")
+        CRITICAL,
+
+        @JsonProperty("important")
+        IMPORTANT,
+
+        @JsonProperty("others")
```

Comment:
shouldn't this be `other`?

Reply:
In the playbook it is "others"

Reply:
Thanks for this finding, I've fixed it in playbook. "Others" is not correct and not aligned with singular name of "important" and "critical"

Reply:
Ok, will fix this too.